db/src/Pos/DB/Block/Epoch.hs

```diff
@@ -286,6 +403,7 @@ consolidateEpochBlocks fpath xs = ExceptT $ do
                         ]
                 pure . Right $
                     SlotIndexLength (getSlotIndex lsi)
+                        -- 12 == sizeof "blnd" + 2 * sizeof Word32
```

Comment:
I would rather see this as as a `let` binding to  prevent it from going out of sync than as a magic number, but not enough to block the PR.